core/core-main/src/main/java/info/nightscout/androidaps/extensions/TemporaryBasalExtension.kt

```diff
@@ -60,6 +60,13 @@ fun TemporaryBasal.toStringShort(): String =
     if (isAbsolute || type == TemporaryBasal.Type.FAKE_EXTENDED) DecimalFormatter.to2Decimal(rate) + "U/h"
     else "${DecimalFormatter.to0Decimal(rate)}%"
 
+fun TemporaryBasal.toStringShort(forcePercentage: Boolean, profileBasal: Double): String =
+    if (isAbsolute || type == TemporaryBasal.Type.FAKE_EXTENDED) {
+        if (forcePercentage) to0Decimal(rate / profileBasal * 100) + "%"
+        else to2Decimal(rate) + "U/h"
```

Comment:
Units should be localized string

Reply:
as you can see - I did not changed its representation. it already was 'U/h' there, I only added '%' variant and it does not need localization. so if this string must become localized - that would me made via additional commit, not this one.

Reply:
<img width="907" alt="image" src="https://user-images.githubusercontent.com/4157587/204521713-157df63d-639c-4d85-90eb-24b06c69df70.png">

Reply:
No problem, it does not hurt to fix it now. If you want we can do it later.